service/templates/admin.html

```diff
@@ -1893,52 +1893,58 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										<textarea ng-model="Settings.providers.aws.allowed_instance_types"
```

Comment:
GitHub doesn't let me comment on unchanged lines but L1891 needs to have `width:45%; margin-left:50px;` to preserve the consistency of adming settings styling.